### ISSUE TITLE
rxjs-no-unsafe-scope not working with instanceof

### ISSUE THREAD

**Issue by kctang** [cartant/rxjs-tslint-rules#67]
Some usage of `instanceof` will cause false positive for this rule.

See example below:

```typescript
// (put this methods in a class)

  instanceof_notOk () {
    return of(1).pipe(
      concatMap((val: any) => {
        // Unsafe scopes are forbidden
        // "rxjs-no-unsafe-scope": {
        //   "options": [
        //     {
        //       "allowDo": false,
        //       "allowMethods": true,
        //       "allowParameters": true,
        //       "allowProperties": true,
        //       "allowTap": false
        //     }
        //   ]
        // }
        return of(val instanceof Array)
      })
    )
  }

  instanceof_ok (val: any) {
    return of(val instanceof Array)
  }
```

**Comment by cartant** [comment]
This should be fixed in 4.14.1.
